libs/crowdfunding/editor/src/lib/crowdfunding-goal-list.tsx

```
@@ -0,0 +1,76 @@
+
+  return (
+    <>
+      <Grid fluid>
+        <Row>
+          <Col xs={5}>{t('crowdfunding.goalsForm.title')}</Col>
+          <Col xs={5}>{t('crowdfunding.goalsForm.description')}</Col>
+          <Col xs={5}>{t('crowdfunding.goalsForm.amount')}</Col>
+          <Col xs={4}>{t('crowdfunding.goalsForm.remove')}</Col>
+        </Row>
+        {goals.map((goal, index) => (
+          <Row>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)